packages/base/Dropdown/src/Dropdown/styles.ts

```diff
@@ -1,58 +1,7 @@
-import type { Theme } from '@material-ui/core/styles'
-import { createStyles } from '@material-ui/core/styles'
-
-export type StyleProps = {
-  /** Control content element style */
-  contentStyle?: {
-    height?: string
-    maxHeight?: string
-  }
+export const contentClass: Record<'content' | 'contentVisible', string> = {
+  content: `overflow-y-auto max-h-[14.75rem]
+  [@media(max-height:585px)]:max-h-[calc(50vh-3rem)]
+  [@media(max-height:585px)]:md:max-h-[calc(50vh-3.5rem)]`,
+  contentVisible: `max-h-screen overflow-y-scroll md:max-h-none md:overflow-y-hidden
+    [@media(max-height:585px)]:max-h-screen overflow-y-scroll`,
```

Comment:
```suggestion
export const contentClass: Record<'content' | 'contentVisible', string> = {
  content: [
    'overflow-y-auto',
  
    'max-h-[14.75rem]',
    '[@media(max-height:585px)]:max-h-[calc(50vh-3rem)]',
    '[@media(max-height:585px)]:md:max-h-[calc(50vh-3.5rem)]',
  ],
  contentVisible: [
    'max-h-screen  md:max-h-none ',
    
    'overflow-y-scroll md:overflow-y-hidden',
    
    '[@media(max-height:585px)]:max-h-screen',
    '[@media(max-height:585px)]:overflow-y-scroll',
  ],
```

NIT: for me personally this way it is easier to read 🤔